src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Models/PSManagementGroupNoChildren.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.ResourceManager.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Models
+{
+    public class PSManagementGroupNoChildren
```

Comment:
The issue is with output formatting. Children are shown only when -Expand is supplied. When it is not, the output has Children with null (even though the group may have children). This might give an impression to the user that the group has no children and to avoid this behavior I added a new object which does not have children at all.